### PR TITLE
Owl iri as a primary key

### PR DESCRIPTION
**Pull request by kasper-piskorski**

### CODE REVIEW

File: mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java [hunk 1]
```diff
@@ -139,7 +170,7 @@ public Entity entity(OWLNamedIndividual individual) {
                     .findFirst();
             return expr.isPresent() ? expr.get().getClassExpression().asOWLClass() : null;
         });
-        return graph.putEntity(id, owlclass == null ? owlThingEntityType() : entityType(owlclass));
+        return putEntity(id, owlclass == null ? owlThingEntityType() : entityType(owlclass));
```
Comment (fppt) [inline mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java]: This putEntity needs to become addEntity at some point.

Comment (kasper-piskorski) [inline mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java]: you mean name-wise? cause it's not a function from the graph api

Comment (fppt) [inline mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java]: Doesn't `putEntity` end up calling `graph.putEntity` at some point ? If not then don't worry about it. You have already repaired the issue.

Comment (fppt) [inline mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OWLMigrator.java]: In that case nevermind my comment and awesome !

**Pull request merged**